stripe/_stripe_client.py

```diff
@@ -280,3 +284,62 @@ def construct_event(
         )
 
         return event
+
+    def raw_request(self, method_: str, url_: str, **params):
+        params = params.copy()
+        options, params = extract_options_from_dict(params)
+        api_mode = params.pop("api_mode", None)
+        base_address = params.pop("base", "api")
+
+        stripe_context = params.pop("stripe_context", None)
+
+        # stripe-context goes *here* and not in api_requestor. Properties
+        # go on api_requestor when you want them to persist onto requests
+        # made when you call instance methods on APIResources that come from
+        # the first request. No need for that here, as we aren't deserializing APIResources
+        if stripe_context is not None:
+            options["headers"] = options.get("headers", {})
+            assert isinstance(options["headers"], dict)
+            options["headers"].update({"Stripe-Context": stripe_context})
+
+        rbody, rcode, rheaders = self._requestor.request_raw(
+            method_,
+            url_,
+            params=params,
+            options=options,
+            base_address=base_address,
+            api_mode=api_mode,
+            usage=["raw_request"],
+        )
+
+        return self._requestor._interpret_response(rbody, rcode, rheaders)
+
+    async def raw_request_async(self, method_: str, url_: str, **params):
+        params = params.copy()
+        options, params = extract_options_from_dict(params)
+        api_mode = params.pop("api_mode", None)
```

Comment:
We won't need to pass api_mode for raw_request because this will be calculated automatically from the path by a util method in api_requestor. 

I think that was not added on the [request_raw method](https://github.com/stripe/stripe-python-next/blob/015a4930c0b90859cc20ed9416fa6e9621b639b9/stripe/_api_requestor.py#L670) but exists on the [request method](https://github.com/stripe/stripe-python-next/blob/015a4930c0b90859cc20ed9416fa6e9621b639b9/stripe/_api_requestor.py#L185).

Reply:
good catch!  this is correct; will update this shortly

Reply:
resolved in https://github.com/stripe/stripe-python/pull/1399